rosbag2_compression/src/rosbag2_compression/zstd_compressor.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdio>
+
+#include <chrono>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "rosbag2_storage/filesystem_helper.hpp"
+
+#include "logging.hpp"
+#include "rosbag2_compression/zstd_compressor.hpp"
```

Comment:
alpha sort

Reply:
`rosbag2_storage` is technically a different project no?

Reply:
The right order would be:

```
#include "rosbag2_compression/zstd_compressor.hpp"

#include "rosbag2_storage/filesystem_helper.hpp"

#include "logging.hpp"
```